docs/cookbook/presigned-get-download-from-browser.md

```diff
@@ -0,0 +1,51 @@
+# Using pre-signed URLs to download via the browser [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/minio/minio?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+Instead of using an S3-compatible client to download files, you can pre-sign a
+request to download a specified file from the browser, even if its bucket is
+private. The following is an annotated example of how this can be used in a
+Node.js application, using [minio-js](https://github.com/minio/minio-js).
+
```

Comment:
you can say:
Using presigned URLs you can allow Web-browser to download a private file directly from S3 without exposing S3 credentials to the browser.